opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicExporters.java

```diff
@@ -41,7 +41,7 @@ public static void start(Configuration configuration) {
         BatchSpanProcessor.newBuilder(spanExporterBuilder.build())
             .setScheduleDelayMillis(configuration.collectionIntervalSeconds * 1000)
             .build();
-    OpenTelemetrySdk.getTracerProvider().addSpanProcessor(spanProcessor);
+    OpenTelemetrySdk.getTracerManagement().addSpanProcessor(spanProcessor);
```

Comment:
Is the readme out of date at this point? And if so, can we update it as part of this PR?

https://github.com/newrelic/opentelemetry-exporter-java#for-spans

For example the readme still says to use:

```
    TracerSdkProvider tracerSdkProvider = (TracerSdkProvider) OpenTelemetry.getTracerProvider();
    tracerSdkProvider.addSpanProcessor(spanProcessor);
```

However I was getting a classcastexception. Doing it the way I see it here seems to resolve the issue. Looks like our examples in the readme are probably just using outdated APIs.

```
TracerSdkManagement tracerSdkManagement = OpenTelemetrySdk.getTracerManagement();
    tracerSdkManagement.addSpanProcessor(spanProcessor);
```

Reply:
Oh 💩 yes that is a good call.  I will change the readme to reflect contemporary reality.